addons/sms/static/src/xml/thread.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<template id="template" xml:space="preserve">
+
+     <t t-extend="mail.widget.Thread.Message">
+        <t t-jquery=".o_thread_tooltip_container" t-operation="after">
+            <span t-if="message.getType() === 'sms' and options.displaySmsIcons" class="o_thread_sms_tooltip_container">
+                <t t-set="thread_icon_class" t-value="'o_thread_sms_tooltip o_thread_message_sms o_thread_message_sms_' + message.getSmsStatus()" />
+                <t t-if="message.isError()">
+                    <t t-set="thread_icon_class" t-value="thread_icon_class + ' o_thread_message_sms_error'"/>
+                </t>
+                <i t-att-class="thread_icon_class + ' fa fa-comment'" t-att-data-message-id="message.getID()"/>
```

Comment:
why not use `t-attf-class` instead of `t-set` and `t-att-class`?

example:

```xml
<i t-attf-class="o_thread_sms_tooltip o_thread_message_sms o_thread_message_sms_#{message.getSmsStatus()} #{message.isError() ? 'o_thread_message_sms_error' : ''} fa fa-comment'"
   t-att-data-message-id="message.getID()"/>
```